mobile/src/main/java/ch/epfl/sdp/mobile/application/tournaments/TournamentFacade.kt

```diff
@@ -0,0 +1,52 @@
+package ch.epfl.sdp.mobile.application.tournaments
+
+import ch.epfl.sdp.mobile.application.authentication.AuthenticatedUser
+import ch.epfl.sdp.mobile.infrastructure.persistence.auth.Auth
+import ch.epfl.sdp.mobile.infrastructure.persistence.store.Store
+
+/**
+ * An interface which represents all the endpoints and available features for tournaments.
+ *
+ * @param auth the [Auth] instance which will be used to handle authentication.
+ * @param store the [Store] which is used to manage documents.
+ */
+class TournamentFacade(private val auth: Auth, private val store: Store) {
```

Comment:
Question: Why not create dummy functions ?

Reply:
Mainly because they are not really testable (since they do nothing), and might lower the coverage below 80%.

Reply:
Can your add TODO keyword somewhere and add a what you explain here inside the code ?

Reply:
I can, but I thougth explaining it in the PR might be better (as this is what was discussed during the Tournament meeting).
Fixed in https://github.com/epfl-SDP/android/pull/321/commits/d812f567f5999a8a38791dbbd9ee8f0b14205ce4.